daisy/test_data/test.wf.json

```diff
@@ -77,15 +77,29 @@
         {
           "Name": "image-from-local-disk",
           "SourceDisk": "local-image",
-          "StorageLocations": ["europe-west1"]
+          "StorageLocations": ["europe-west1"],
+          "Description": "Some Ubuntu",
+          "Family": "ubuntu-1404",
+          "Project": "a_project",
+          "NoCleanup": true,
+          "ExactName": false,
+          "OverWrite": false,
+          "GuestOsFeatures": ["VIRTIO_SCSI_MULTIQUEUE", "UBUNTU", "MULTI_IP_SUBNET"]
         }
       ]
     },
     "create-image": {
       "createImages": [
         {
           "Name": "image-from-disk",
-          "SourceDisk": "image"
+          "SourceDisk": "image",
+          "Description": "Microsoft, SQL Server 2016 Web, on Windows Server 2019",
+          "Family": "sql-web-2016-win-2019",
+          "Project": "a_project",
+          "NoCleanup": true,
```

Comment:
Is "NoCleanup" required here? It shouldn't be required if validation is done in the workflow.

Reply:
This is testing workflow parsing is actually reading all the fields. Workflow itself will not be run here.